frontend/apps/reader/modules/readerview.lua

```diff
@@ -1022,14 +1026,40 @@ function ReaderView:onToggleReadingOrder()
 end
 
 function ReaderView:getTapZones()
-    local forward_zone = {
-        ratio_x = DTAP_ZONE_FORWARD.x, ratio_y = DTAP_ZONE_FORWARD.y,
-        ratio_w = DTAP_ZONE_FORWARD.w, ratio_h = DTAP_ZONE_FORWARD.h,
-    }
-    local backward_zone = {
-        ratio_x = DTAP_ZONE_BACKWARD.x, ratio_y = DTAP_ZONE_BACKWARD.y,
-        ratio_w = DTAP_ZONE_BACKWARD.w, ratio_h = DTAP_ZONE_BACKWARD.h,
-    }
+    local forward_zone, backward_zone
+    local tap_zones_type = G_reader_settings:readSetting("page_turns_tap_zones", "default")
+    if tap_zones_type == "default" then
+        forward_zone = {
+            ratio_x = DTAP_ZONE_FORWARD.x, ratio_y = DTAP_ZONE_FORWARD.y,
+            ratio_w = DTAP_ZONE_FORWARD.w, ratio_h = DTAP_ZONE_FORWARD.h,
+        }
+        backward_zone = {
+            ratio_x = DTAP_ZONE_BACKWARD.x, ratio_y = DTAP_ZONE_BACKWARD.y,
+            ratio_w = DTAP_ZONE_BACKWARD.w, ratio_h = DTAP_ZONE_BACKWARD.h,
+        }
+    else -- user defined page turns tap zones
+        local tap_zone_forward_w = G_reader_settings:readSetting("page_turns_tap_zone_forward_size", DTAP_ZONE_FORWARD.w)
```

Comment:
`"page_turns_tap_zone_forward_screen_ratio"` ?
So we don't read `page_turns_tap_zone_forward_size = 75` as it means 75px.